src/run.ts

```diff
@@ -17,6 +17,15 @@ export default async function(helper: Helper, args: CreateAppArgs) {
 	const isSkeleton = args.skeleton;
 	const isTsx = args.tsx;
 
+	const scripts = {
+		dev: 'dojo build --mode dev --watch file --serve',
+		build: 'dojo build --mode dist',
+		buildDev: `dojo build --mode dev`,
+		test: 'npm run test:unit && npm run test:functional',
```

Comment:
I think that `npm run test` should translate to `dojo test` that doesn't need a bundle to be built as it does transpilation on the fly.

Perhaps the others:
 * `npm run test:unit` should be `dojo build --mode unit && dojo test --unit --config local` 
 * `npm run test:functional` should be `dojo build --mode functional && dojo test --functional --config local` 
 * `npm run test:all` should be `dojo build --mode unit && dojo build --mode functional && dojo test --all --config local`